tests/code/test_events.py

```diff
@@ -34,7 +34,8 @@
 @pytest.mark.integration
 def test_event_model():
     event = Event.get_by_id(11)
-    assert event.isPast
+    assert event.isPastStart
+    assert event.isPastEnd
```

Comment:
We should beef up this test a little bit. Testing for the negative cases, and where an event has started and not finished yet.